config/OptionsBlocklist.lua

```diff
@@ -222,9 +222,81 @@ local function CreateNPCBlocklistOptions()
 	end)
 end
 
+local function CreateTitleBlocklistOptions()
+	local panel = CreateOptionsPanel('TitleBlocklist',
+		L['Title Blocklist'],
+		L['Quests with titles that partially match or IDs that exactly match entries from this list will not be automated.'],
+		L['Block Title'])
```

Comment:
These three should be added as commented out lines in the various localization files.